Khan/wonder-blocks
# Start creating component gallery (sticker sheet) on Wonder Blocks overview page

## Summary:
We want to add a view to the overview page that shows all available
Wonder Blocks components in one location.

I took some design inspiration from the [Material UI gallery](https://m3.material.io/components).

SOME NOTES:

1. The rest of the descriptions will be added in the following PR. This one just includes one
description (Accordion) to demonstrate what that looks like.
2. I made the tiles open the pages in a new tab so that they wouldn't
open inside the iframe.
3. I think there is something wrong with the keyboard nav for the tile headers in Firefox and Safari, but I'm not sure why.
4. There are a few things I wasn't sure if I should add, so I left them out for now:

- Clickable
- Core
- Data
- Layout
- Translations
- Modal building blocks
- Testing
- Timing
- Theming
- Typography



Issue: https://khanacademy.atlassian.net/browse/WB-1595

## Test plan:
- Go to http://localhost:6061/?path=/docs/overview--docs
- Confirm that the tiles look okay (doesn't matter how the actual components look for now).
- Confirm that the hover/focus styles works.
- Confirm that the tiles link to the correct pages.
